src/main/fc/core.c

```diff
@@ -422,7 +423,7 @@ void updateArmingStatus(void)
     }
 }
 
-void disarm(void)
+void disarm(uint32_t reason)
```

Comment:
Should we pass in a `FlightLogDisarmReason` enum here?

Reply:
yes I think so, I think I forgot to go back and change this.  Will fix.